src/system.rs

```diff
@@ -1,8 +1,8 @@
-//! Provides functions for instantiating instances of [Schema] using [SchemaSystem].
+//! Provides functions for instantiating instances of [`Schema`] using [`SchemaSystem`].
```

Comment:
```suggestion
//! Provides functions for getting instances of [`Schema`] using [`SchemaSystem`].
```
"instantiating instances" is repetitive.